pydata_center/monitoring/exceptions.py

```diff
@@ -0,0 +1,44 @@
+from typing import Union
+
+from rest_framework import status
+
+
+class MonitoringBaseException(Exception):
+    status_code: int
+    message: str
+    detail_info: Union[str, bool] = False
+
+    def __init__(
+            self,
+            message: str = None,
+            detail_info: Union[str, bool] = False
+    ):
+        if message:
+            self.message = message
+        self.detail_info = detail_info
+        super().__init__(self.message)
```

Comment:
Setting `message` manually is redundant. If you just want an additional `detail_info` attribute, you could do

```python
def __init__(self, message: str = None, detail_info: Union[str, bool] = False):
    super().__init__(message)
    self.detail_info = detail_info
```

Also, do we really need `detail_info`? The default `Exception` already allows us to pass multiple arguments, one of which can serve the same purpose.